html/generatedReport.php

```diff
@@ -75,61 +77,69 @@ function generateReport($site) {
 		$beginDate = new DateTime ( date ( DATE_ATOM, mktime ( 0, 0, 0, $endMonth - 3, 01, date ( "Y" ) - 1 ) ) );
 		$beginDate = $beginDate->format ( "Y-m-d" );
 	}
-	
+
 	// If its an authenticated user, then only continue to generate the report.
 	if (! authenticatedUser ()) {
-		unset ( $_SESSION ['submitReport'] );
+		unset ( $GLOBALS ['submitReport'] );
 		return "invalidLogin";
 	}
 	deleteFilesInDownloadDir();
 	$quarter = determineFourQuarters ( substr ( $endDate, 5, 2 ), $_POST ['quarter'] );
-	
+
 	// First compute all the 4 quarters thats necessary to generate the report.
-	
+
 	// Include the file that is used to compute the total number of packages and compute it
 	require_once ('totalNumberOfDataPackages.php');
 	createTotalDataPackagesInputData ( $beginDate, $endDate );
-	if (isset ( $_SESSION ['totalDataPackages'] ) && $_SESSION ['totalDataPackages'] != null) {
+	if (isset ( $GLOBALS ['totalDataPackages'] ) && $GLOBALS ['totalDataPackages'] != null) {
 		$deleteCount = countDeletedPackages ( $beginDate, $endDate, $quarter , $site);
-		createTotalDataPackagesOutput ( $_SESSION ['totalDataPackages'], $quarter, $deleteCount, $site );
+		createTotalDataPackagesOutput ( $GLOBALS ['totalDataPackages'], $quarter, $deleteCount, $site );
+    $GLOBALS ['totalDataPackages'] = -1;
 	}
-	// Adding a sleep command as making numerous calls to PASTA in a short interval results in failure to get the information.
-	sleep ( 2 );
-	
+
+  // Adding a sleep command as making numerous calls to PASTA in a short interval results in failure to get the information.
+	sleep ( 4 );
```

Comment:
from 2 to 4 seconds of sleep, why did we change that? Were we calling PASTA too frequently?